ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1450,6 +1450,10 @@ tests:
       E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @arm64 and @network-ovnkubernetes
         and not @fips'
       TAG_VERSION: '@4.11'
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OVNKubernetes
+        FIPS_MODE=true
```

Comment:
The job name needs update to reflect it's testing fips, also the test filter needs update